src/main/java/frc/robot/subsystems/DriveBaseSubsystem.java

```diff
@@ -54,18 +59,26 @@ public void periodic() {
 
   // Normal Arcade Drive
   public void arcadeDrive() {
-    m_differentialDrive.arcadeDrive( m_driverJoystick.getRawAxis(Constants.kDBLeftJoystickAxisY), m_driverJoystick.getRawAxis(Constants.kDBRightJoystickAxisY));
+    m_differentialDrive.arcadeDrive( m_driverJoystick.getRawAxis(Constants.kDBLeftJoystickAxisY), 
+                                      m_driverJoystick.getRawAxis(Constants.kDBRightJoystickAxisY));
 
   }
 
+  // Arcade Drive where you can only move forwards and backwards for testing
+  public void arcadeDrive(double rotation) {
```

Comment:
If you want to use these different modes, you'll need to setup a command to switch  to them or add something to Shuffleboard for them.